priv/templates/phx.gen.auth/session_controller.ex

```diff
@@ -14,6 +14,7 @@ defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web
     if <%= schema.singular %> = <%= inspect context.alias %>.get_<%= schema.singular %>_by_email_and_password(email, password) do
       <%= inspect schema.alias %>Auth.log_in_<%= schema.singular %>(conn, <%= schema.singular %>, <%= schema.singular %>_params)
     else
+      # In order to prevent user enumeration attacks, don't discolose whether the email is registered.
```

Comment:
Typo:

```suggestion
      # In order to prevent user enumeration attacks, don't disclose whether the email is registered.
```

Reply:
@josevalim @aaronrenner should I fix the same typo in https://github.com/aaronrenner/phx_gen_auth/ or is that repo to be retired?

Reply:
Let's fix the typo in phx.gen.auth. That repo will be maintained until 1.6 is out.